src/experimental/build/commands/build/get_context.ts

```diff
@@ -15,7 +15,14 @@ export async function getContext(
 ): Promise<Context> {
     const stableContext = await getStableContext(canisterName, {
         ...canisterConfig,
-        custom: { ...canisterConfig.custom, openValueSharing: undefined }
+        custom: {
+            ...canisterConfig.custom,
+            openValueSharing: undefined,
```

Comment:
Can you put these in alphabetical order?

Reply:
Done